core/domain/exp_domain.py

```diff
@@ -3526,6 +3526,9 @@ def _convert_states_v11_dict_to_v12_dict(cls, states_dict):
                 }
                 for old_translation in old_audio_translations
             }
+            old_solution = state_dict['interaction']['solution']
```

Comment:
(Optional) This is a _little_ convoluted. Perhaps cleaner to do:

```
    if not state_dict['interaction']['solution']:
        state_dict['interaction']['solution'] = None
```

Please also update the docstring. Also, I'm fairly sure you'll need to fix some tests...